internal/helper/helper_darwin.go

```diff
@@ -0,0 +1,23 @@
+package helper
+
+import (
+	"github.com/ozansz/gls/internal"
+	"io/fs"
+	"syscall"
+)
+
+func (fsInfo FsInfo) GetSize(fInfo fs.FileInfo) (int64, error) {
+	st, ok := fInfo.Sys().(*syscall.Stat_t)
+	if !ok {
+		return nil, fmt.Errorf("could not cast %T to syscall.Stat_t", fInfo.Sys())
```

Comment:
the `nil` returns here should be numeric (function return signature is `(int64, error)`), let's use `0` instead of `nil`